src/PHPMailer.php

```diff
@@ -3456,10 +3457,10 @@ public function encodeHeader($str, $position = 'text')
                 }
                 $matchcount = preg_match_all('/[^\040\041\043-\133\135-\176]/', $str, $matches);
                 break;
-            /* @noinspection PhpMissingBreakStatementInspection */
+                /* @noinspection PhpMissingBreakStatementInspection */
```

Comment:
Most of these case statement indent changes are incorrect. The comment applies to the case statement below it, not to the indented one above